jime.csl

```diff
@@ -0,0 +1,362 @@
+      <et-al/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+        <names variable="director"/>
+        <choose>
+          <if type="bill book graphic legal_case legislation motion_picture report song webpage" match="any">
+            <text variable="title" form="short" font-style="italic"/>
+          </if>
+          <else>
+            <text variable="title" form="short" quotes="true" text-case="capitalize-first"/>
+          </else>
+        </choose>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="date-issued">
+    <choose>
+      <if type="article-journal interview" match="none">
+        <text macro="date-issued-year"/>
+      </if>
+      <else-if type="interview" match="any">
+        <text macro="date-issue-long"/>
+      </else-if>
+      <else-if variable="page" match="none">
+        <text macro="date-issued-year"/>
+      </else-if>
+      <else-if is-numeric="page">
+        <text macro="date-issued-year"/>
+      </else-if>
+      <else>
+        <text term="in press" prefix="(" suffix=")"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="date-issued-no-parentheses">
+    <choose>
+      <if type="article-journal" match="none">
+        <text macro="date-issued-year"/>
+      </if>
+      <else-if variable="page" match="none">
+        <text macro="date-issued-year"/>
+      </else-if>
+      <else-if is-numeric="page">
+        <text macro="date-issued-year"/>
+      </else-if>
+      <else>
+        <text term="in press"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="date-issued-year">
+    <choose>
+      <if variable="issued">
+        <date date-parts="year" form="numeric" variable="issued">
+          <date-part name="year"/>
+        </date>
+      </if>
+      <else>
+        <text term="no date" form="short"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="date-access">
+    <date variable="accessed" prefix="[Last accessed " suffix="]">
+      <date-part name="day" form="numeric-leading-zeros" prefix=""/>
+      <date-part name="month" form="long" prefix=" "/>
+      <date-part name="year" prefix=" "/>
+    </date>
+  </macro>
+  <macro name="title">
+    <choose>
+      <if type="bill book graphic legal_case legislation motion_picture report song webpage broadcast" match="any">
+        <text variable="title" font-style="italic"/>
+      </if>
+      <else>
+        <text variable="title"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="book-details">
+    <group delimiter=". ">
+      <group delimiter=" ">
+        <number vertical-align="baseline" variable="edition" form="ordinal"/>
+        <label vertical-align="baseline" variable="edition" form="short"/>
+      </group>
+      <group delimiter=" ">
+        <label text-case="capitalize-first" variable="volume"/>
+        <text variable="volume"/>
+      </group>
+      <group font-variant="normal">
```

Comment:
Please delete any ` font-variant="normal"` throughout the style unless it's on purpose.

Reply:
Any use of font-variant="normal" came from the template CSL used to start this file. I have removed references to it via 482b205, including reducing lines 153-155 by removing what seems like an unnecessary group to just have one line to call the editor-translator macro.